api/config/__init__.py

```diff
@@ -24,13 +25,14 @@ def __init__(self, **data: Any) -> None:
         super().__init__(**data)
         if self.testing:
             self.database_name = f"{self.database_name}"
+        self.hydroshare_meta_read_url = str(self.hydroshare_meta_read_url)
+        self.hydroshare_file_read_url = str(self.hydroshare_file_read_url)
 
     @property
     def db_connection_string(self):
         return f"{self.db_protocol}://{self.db_username}:{self.db_password}@{self.db_host}/?retryWrites=true&w=majority"
 
-    class Config:
-        env_file = ".env"
+    model_config = SettingsConfigDict(env_file=".env", env_file_encoding="utf-8")
```

Comment:
Just noting that I have since learned that if we leave the env_file Config out then it just reads environment variables which eases deployments. This comment is not relevant to these changes, just making a note here.